dns.tf

```
@@ -0,0 +1,16 @@
+resource "google_dns_managed_zone" "default" {
+  project = var.project
+  name        = "iamjustintime-livelinklabs-delegation"
+  dns_name    = "${var.dns_name}."
+  description = "Delegated zone for iamjustintime application"
+}
+
+resource "google_dns_record_set" "default" {
+  project = var.project
+  name = google_dns_managed_zone.default.dns_name
+  type = "A"
+  ttl  = 300
```

I guess 24 hours is OK for us 👌 We always can downscale it to 1 min if there will be a need to re-create Cloud Run deployment. Remember, we pay for each DNS request in the cloud.
```suggestion
  ttl  = 86400
```